src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -1564,7 +1563,7 @@ public static void JsonContainingOnlyWhitespaceIsInvalid()
 
         [Theory]
         [InlineData("//", 2)]
-        [InlineData("//\n", 3)]
+        [InlineData("//\n", 0)]
```

Comment:
Can you add Assert.Equal(expectedLineNumber, ex.LineNumber) as well for all these tests.

For most of them, `expectedLineNumber` should be 0, except for this one, where it should be 1.